profile/README.md

```diff
@@ -0,0 +1,63 @@
+# Lombiq Technologies 
+
+
+[Lombiq Technologies](https://lombiq.com) is a software, training, and services company focusing on web development with open Microsoft technologies. 
+Our main area of expertise is the Orchard Core CMS. We provide custom development, training, hosting, and consulting services and we are behind the first (and only) Orchard CMS SaaS.
+
+## Products:
+
+* [**Hastlayer**](https://hastlayer.com/) turns performance-critical sections of .NET programs into computer chips. You can still update your software, but it will be a whole lot faster and more power-efficient.
+![Hastlayer-logo](https://user-images.githubusercontent.com/68704777/171685373-7a64e9bf-b0fc-431d-a8d1-9027e0989b15.png)
+
+* [**DotNest** ](https://dotnest.com/) We've developed DotNest as a software-as-a-service for Orchard CMS, enabling users to create their websites using Orchard in a flash. Today it runs more than 6000 websites.
+![er6XI9B](https://user-images.githubusercontent.com/68704777/171690190-225d15a4-412d-40f4-b97f-63e192b4e596.png)
+
+## Our services:
+
+### Software development:
+
+Our team’s passion is developing software with
+care for clients who need help with their most
+extraordinary projects in the following frameworks:
+
+Orchard Core \
+Orchard CMS \
+ASP.NET Core \
+ASP.NET
```

Comment:
Did you intentionally avoid making this a [list](https://www.markdownguide.org/basic-syntax/#unordered-lists)?